### DISCUSSION TITLE
Welcome to GatorGrader Discussions!

### DISCUSSION THREAD

**Discussion by Michionlion** [GatorEducator/gatorgrader#224] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by blackeuler** [comment]
### 👋🏿 HI Im Chris

Im currently a software engineer @ Corvus Insurance slinging lines of Elixir and Elm.
I have been looking at contributing more seriously to GatorGrader over the winter break but alas Life. 
I wish to help make GatorGrader a tool that continues to help faculty help  students @ Allegheny College
 learn the joys of computation and software development.
 
 And maybe explore the applications to non-technical courses.
